core/types.go

```diff
@@ -32,6 +32,17 @@ type Validator interface {
 	// ValidateState validates the given statedb and optionally the receipts and
 	// gas used.
 	ValidateState(block *types.Block, state *state.StateDB, receipts types.Receipts, usedGas uint64) error
+
+	// StartRemoteVerify start the remote verify routine for given peers.
+	StartRemoteVerify(peers VerifyPeers)
```

Comment:
I think there is some way to hide `StartRemoteVerify` and `StopRemoteVerify`, we should expose as less interface as possible.